### PR TITLE
Fix bad PhysicalShapeLayer performance when using system_composite

### PR DESCRIPTION
**Pull request by arbreng**

### CODE REVIEW

**Comment by arbreng** [comment]
the parts I deleted were left over from a bad merge in https://github.com/flutter/engine/pull/14577

File: flow/layers/physical_shape_layer.cc [hunk 1]
```diff
@@ -61,57 +59,26 @@ void PhysicalShapeLayer::Preroll(PrerollContext* context,
     set_paint_bounds(path_.getBounds());
   } else {
     if (PhysicalShapeLayerBase::can_system_composite()) {
+      // always system composite these layers when possible and an elevation is
```
Comment (gaaclarke) [inline flow/layers/physical_shape_layer.cc]: Comments should be proper prose: https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar

**Comment by gaaclarke** [comment]
Can you supply some data to justify the change?  Maybe a performance test?  Maybe a skia trace?

**Comment by iskakaushik** [comment]
@gaaclarke fxb/43364 has traces relating to this regression

**Comment by arbreng** [comment]
Unfortunately this ended up not being a fix in all situations, based on traces in the linked fxb/ issue.

Going to make another PR with a targeted revert of the offending code.

**Pull request closed without merging**